package.json

```diff
@@ -146,7 +146,6 @@
     "@types/react": "17.0.14",
     "@types/react-dom": "17.0.14",
     "@types/react-linkify": "1.0.1",
-    "@types/react-native": "0.69.22",
```

Comment:
This package has been deprecated.

Reply:
Are the types bundled with the react-native package now?

Reply:
Updating to "@types/react-native": "0.73.0" logs a warning that react-native provides its own type definitions, so we don't need @types/react-native installed anymore.